### DISCUSSION TITLE
Where is the ./configure script?

### DISCUSSION THREAD

**Discussion by danielchen0** [inspircd/inspircd#2014] [Support]
**Description**

I installed inspIRCd via the instructions here: https://docs.inspircd.org/3/installation/ubuntu/

I have been having issues with SSL not being enabled, so I've been following the instructions here: https://docs.inspircd.org/faq/#why-does-my-server-tell-me-an-unexpected-tls-packet-was-received-or-error1408f10bssl-routinesssl3_get_recordwrong-version-numbersslrecordssl3_recordc-when-i-try-to-connect-using-ssltls

**Steps to reproduce the issue:**
I have installed OpenSSL from here: https://www.openssl.org/source/

Now I get to here: https://docs.inspircd.org/3/modules/ssl_openssl/
Which suggests this instruction:
```./configure --enable-extras ssl_openssl```

I don't know where the configure script is.

inspIRCd is unable to find the .so if I add this to the conf file:
```<module name="ssl_openssl">```

**Describe the results you received:**
No configure script

**Describe the results you expected:**
Some configure script

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `./bin/inspircd --version`:**

InspIRCd-3.15.0-deb2

**Comment by sadiepowell** [comment]
Those are the instructions for building from source. If you're using a package those instructions do not apply.

As you're using our packages you should use the [ssl_gnutls](https://docs.inspircd.org/3/modules/ssl_gnutls) module instead of ssl_openssl as our packages do not ship with the latter.

**Reply by danielchen0** [reply]
Thanks for the reply, I'm still confused as to how to enable the ssl_gnutls module for the package - the instructions you linked suggest something similar i.e. use a ./configure script which I do not have

**Reply by sadiepowell** [reply]
I'll update the docs to make it more clear how to do this for package users.

To answer your question for now though:

1. Add a `<module>` tag for the ssl_gnutls module.
2. Add an `<sslprofile>` tag (see the docs page) with your SSL cert/key details.
3. Put the value from `<sslprofile:name>` in a `<bind:sslprofile>` tag.
4. Restart your server (or send it SIGUSR1 if you have the sslrehashsignal module loaded).
